src/systems/variational_smoother_constraint.C

```diff
@@ -342,6 +342,7 @@ void VariationalSmootherConstraint::constrain()
 {
   const auto & mesh = _sys.get_mesh();
   const auto dim = mesh.mesh_dimension();
+  const auto &rank = mesh.comm().rank();
```

Comment:
```suggestion
  const auto &rank = mesh.processor_id();
```

Just to match our usual idioms